src/platform/site-wide/user-nav/components/HelpMenu.jsx

```diff
@@ -33,6 +34,7 @@ class HelpMenu extends React.Component {
 
     return (
       <DropDownPanel
+        onClick={recordEvent({ event: 'nav-jumplink-click' })}
```

Comment:
Hey @ethanteague , I think this should have been wrapped in a function, like -

```
 onClick={() => recordEvent({ event: 'nav-jumplink-click' })}
```

Same with https://github.com/department-of-veterans-affairs/vets-website/pull/10818/files#diff-6ce07a2b29c8f4123001e03206daf27eR58.

Currently, it's called `recordEvent` immediately  (when the component loads) instead of when the event actually fires.